samples/bot-sso-adaptivecard/csharp/BotSsoAdaptivecard/TeamsAppManifest/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.14/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.14",
+  "version": "1.0.0",
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "packageName": "com.microsoft.teams.botssoadaptivecard",
+  "developer": {
+    "name": "Teams App, Inc.",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuse"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "SSO Adaptive Card",
+    "full": "SSO Adaptive Card"
+  },
+  "description": {
+    "short": "Short description for SSO authentication for your Adaptive Cards.",
```

Comment:
Implement SSO authentication for your Adaptive Cards.

Reply:
Fixed